README.md

```diff
@@ -112,6 +112,8 @@ For an overview of the full workflow, see the [workflow diagram](./docs/workflow
 
 3. There are a few ways you can locally install the `ilab` CLI. Select your preferred installation method from the following instructions. You can then install `ilab` and activate your `venv` environment.
 
+   > **NOTE**: ⏳ The `python3` binary shown in the following steps is the Python version that you installed in the above step. The command can be `python3.11` or `python3.10` instead of `python3`. You can check Python's version by `python3 -V`.
```

Comment:
nit: Should this be:

The command can also be ...

to imply python3 is still valid depending on your setup.

Reply:
All right. I fixed the issue on the latest rebase.